src/cleed_core/cleed_core.c

```
@@ -0,0 +1,13 @@
+/*********************************************************************
+ *                       CLEED_CORE.C
+ *
+ *  Core library entry points for CLEED.
+ *********************************************************************/
+
+#include "cleed_core.h"
+#include "search_ver.h"
+
+const char *cleed_core_version(void)
```

Cppcheck (2.19.0)

[style] The function 'cleed_core_version' is never used.

(unusedFunction)